wiki/Tournaments/COE/COE_2025/en.md

```diff
@@ -0,0 +1,230 @@
+
+Interviews are **NOT** mandatory and may be denied by the participant.
+
+The participants in the grand final and the winner of the match for third place are required to attend the prize ceremony.
+
+#### 1.3 Prize Money
+
+The prize money should ideally be paid out 90 days after the tournament has concluded, but may take as long as 180 days for the payment to be completed. If a participant has not provided any payment method to the tournament organisation, the payment will be delayed until this is rectified.
+
+Stichting CAVOE EVENTS reserves the right to cancel any pending payment, if any evidence of fraud or foul play has been discovered.
+
+Stichting CAVOE EVENTS is not liable for any taxes or fees the participant may have to deduct from the prize.
+
+### 2. Requirements
+
+#### 2.1 Game Requirements
+
+Participants are not allowed to use custom skin elements to alter core gameplay elements or mechanics. Any modifications to the game not intended by the developer are also strictly forbidden.
+
+For qualifiers osu!(stable) will be used. For the elimination matches osu!(lazer) will be used.
+
+#### 2.2 Punctuality
+
+Should a participant not be present at the allotted match time they will be given a forfeit loss. Participants are also expected to keep the match running fluently and without intentional delays. Excessive match delays results in penalties being applied at the discretion of the referee.
+
+#### 2.3 Drivers and Setup
+
+During the stages of the tournament that are played in a LAN setting participants are required to inform tournament officials ahead of time which drivers they need for their equipment. Only open source and drivers supplied by the manufacturer are allowed to be used.
+
+Participants will be given the chance before any LAN matches start to configure their drivers and the rest of the setup to their liking.
+
+We reserve the right to inspect any equipment brought by participants and deny it being used, if an attempt at foul play is suspected.
+
+### 3. Qualifiers
+
+#### 3.1 Format Overview
+
+The gamemode being played is osu!standard, all matches will be played using ScoreV2.
+
+During the qualifier, each participant will be given one attempt to set a high score on each map of the map pool. These attempts will be in matches scheduled beforehand.
+
+Each score will be used to determine the seeding. Seeding method used is `x=(y-AVERAGE(z))/STDEV.S(z)` where x is the point result for that map, y is the achieved score on map and z is all scores on map. All x's for a player are added up to form their final qualifier result.
+The top 16 players will advance to the finals.
+
+#### 3.2 Match Procedure
+
+The matches will start once the scheduled time is reached and a referee is present. 
+
+There will be no warm ups during the match. Players get to decide the order.
+A short break is given between each map, the exact duration is at the discretion of the referee.
```

Comment:
dunno about the exact layout, but at least the last two sentences should be in the same paragraph
```suggestion
The matches will start once the scheduled time is reached and a referee is present.  There will be no warm ups during the match. Players get to decide the order. A short break is given between each map, the exact duration is at the discretion of the referee.
```